docs/features/linkerd/tutorials/linkerd-authorization-policies.mdx

```diff
@@ -0,0 +1,234 @@
+---
+sidebar_position: 3
+title: Linkerd AuthorizationPolicy automation
+image: /img/quick-tutorials/k8s-linkerd-authorization-policies/social.png
+---
+
+import CodeBlock from "@theme/CodeBlock";
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+Otterize automates mTLS-based, HTTP-level pod-to-pod access control with Linkerd authorization (authZ) policies, within your Kubernetes cluster.
+
+Implementing this kind of access control with Linkerd can be challenging.
+For example, each target server needs a matching `Server` resource, `MeshTLSAuthentication` resources need to be created & matched with identities
+in order to use mTLS based authorization policies, etc. Using Otterize `ClientIntents`, this is all managed for you.
+
+To help you avoid manually managing complicated authorization policies per server, Otterize implements **intent-based access control** (IBAC).
+You just declare what calls the client pods intend to make,
+and everything is automatically wired together so only intended calls are allowed.
+
+In this tutorial, we will:
+
+- Deploy a Linkerd demo application with two client pods and one server pod.
+- Declare that the first client intends to call the server with a specific HTTP path and method.
+- See that a Linkerd authorization policy was autogenerated to allow just that, and to block the (undeclared) calls from the other client.
+
+## Prerequisites
+
+### 1. Install Linkerd
+Install Linkerd by following the [Installing Linkerd](https://linkerd.io/2-edge/tasks/install/) tutorial.
+
+### 2. Deploy Otterize
+If you do not have a cluster, we will need to prepare one with [network policy support](/overview/installation#create-a-cluster-with-support-for-network-policies)
+
+To deploy Otterize, head over to [Otterize Cloud](https://app.otterize.com) and associate a Kubernetes cluster on the [Integrations page](https://app.otterize.com/integrations), and follow the instructions. If you already have a Kubernetes cluster connected, skip this step.
+
+## Tutorial
+
+### Deploy the two clients and the server
+
+Deploy a simple example consisting of `client` and `other-client` calling `nginx` over HTTP:
+
+```shell
+curl ${ABSOLUTE_URL}/code-examples/linkerd-authorization-policies/all.yaml | linkerd inject - | kubectl apply -f -
+```
+
+Notice we use `linkerd inject` from Linkerd's CLI beforce we apply our resources to Kubernetes, to include them as part of the mesh.
+
+### Apply intents
+
+We will now declare that the **client** intends to call the **server** at a particular HTTP path using a specific HTTP method.
+
+When the intents YAML is applied, creating a custom resource of type `ClientIntents`,
+Otterize will add a Linkerd authorization policy to allow the intended call
```

Comment:
What else will it do? Again, add reference to the rest of the work Otterize does, Server resources, etc.